src/test/javascript/portal/filter/ui/FilterGroupPanelSpec.js

```diff
@@ -283,8 +285,8 @@ describe("Portal.filter.ui.FilterGroupPanel", function() {
         it('creates a new groups as required', function() {
```

Comment:
Maybe just fix the grammer for this test, not a biggie (I know it's not a line you've changed).